app/controllers/application_controller.rb

```diff
@@ -1,2 +1,40 @@
 class ApplicationController < ActionController::Base
+  before_action :set_default_meta_tags
+
+  private
+
+  def set_default_meta_tags
+    title = 'Hotwire.io'
+    description = "Community-driven documentation and resource Hub for Hotwire."
+    image = "#{root_url}og-image.png"
+
+    tags = {
+      site: title,
+      reverse: true,
+      separator: '·',
+      description: description,
+      keywords: ['hotwire', 'stimulus.js', 'turbo drive', 'turbo streams', 'turbo frames', 'documentation', 'community'],
+      twitter: {
+        title: title,
```

Comment:
if we leave out the title here it will automatically take the one we currently set in the `application` layout, so it also includes the title in the share links.